apis/vshn/v1/vshn_nextcloud.go

```diff
@@ -174,6 +178,15 @@ func (v *VSHNNextcloud) SetInstanceNamespaceStatus() {
 	v.Status.InstanceNamespace = v.GetInstanceNamespace()
 }
 
+type CollaboraSpec struct {
+	// Enabled enables the Collabora integration. It will autoconfigure the Collabora server URL in Your Nextcloud instance.
+	//+kubebuilder:default=false
+	Enabled bool `json:"enabled"`
+	// FQDN contains the FQDN of the Collabora server. This is used to configure the Collabora server URL in Your Nextcloud instance.
+	//+kubebuilder:validation:Required
```

Comment:
What happens if you don't specify the `collabora` field at all in a claim? 

Will it be valid, or will it block the creation, because this is required? I'm not sure if the `omitempty` on line 92 is enough for this case, or if we need to explicitly set `//+kubebuilder:validation:Optional` for the `collabora` field.

Reply:
yeah it's enough, I've tested such case and worked pretty well in my setup. I also set:
```
//go:generate yq -i e ../../generated/vshn.appcat.vshn.io_vshnnextclouds.yaml --expression "with(.spec.versions[]; .schema.openAPIV3Schema.properties.spec.properties.parameters.properties.service.collabora.default={})"
```